domains/maurom.json

```
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "MM-coder",
+           "email": "mauromartins@email.com",
+           "discord": "804668335039053874"
+        },
+    
+        "record": {
+            "A": ["185.199.108.153"]
```

For it to work you need to add 3 more A records
```suggestion
        "A": [
            "185.199.108.153",
            "185.199.109.153",
            "185.199.110.153",
            "185.199.111.153"
        ]
```